readme.md

```diff
@@ -290,6 +290,12 @@
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval) - Learn to develop your own search engine.
 
 
+## Natural sciences
+
+- [Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
```

Comment:
You should make 2 separate PR's as per the [guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md#adding-to-this-list)